src/core/SampleBuffer.cpp

```diff
@@ -1140,12 +1134,12 @@ QString & SampleBuffer::toBase64( QString & _dst ) const
 
 
 
-SampleBuffer * SampleBuffer::resample( const sample_rate_t _src_sr,
-						const sample_rate_t _dst_sr )
+std::unique_ptr<SampleBuffer> SampleBuffer::resample(const sample_rate_t _src_sr,
```

Comment:
Why did you declare it as returning `SampleBuffer` in the header? Its instance can't be copied because it's a child of `QObject`.